llvm/lib/DebugInfo/DWARF/DWARFDebugLine.cpp

```diff
@@ -913,6 +916,9 @@ Error DWARFDebugLine::LineTable::parse(
         // followed.
         EmitRow();
         State.resetRowAndSequence();
+        // Cursor now points to right after the end_sequence opcode - so points
+        // to the start of the next sequence - if one exists.
+        LineTableSeqOffset = Cursor.tell();
```

Comment:
We can add a `uint64_t Offset` to `DWARFDebugLine::ParsingState::resetRowAndSequence(uint64_t Offset);` And call this before the while loop.